packages/react-components/src/components/MessageThread.tsx

```diff
@@ -76,7 +76,7 @@ const isMessageSame = (first: ChatMessage, second: ChatMessage): boolean => {
 const getLatestChatMessage = (messages: Message[]): ChatMessage | undefined => {
   for (let i = messages.length - 1; i >= 0; i--) {
     const message = messages[i];
-    if (message.messageType === 'chat' && !!message.createdOn) {
+    if (message?.messageType === 'chat' && !!message.createdOn) {
```

Comment:
Should we also safe guard message?.createdOn?
Or should we return early like this:
if (!message) {
   continue;
}

Reply:
`message` will be guaranteed to be defined when it gets to the `message.createdOn` part of the `&&` because if it was undefined then the first half `message?.messageType === 'chat'` would be false and end the if checks